content/academic_update/waiting/_index.md

```diff
@@ -0,0 +1,74 @@
+
+
+## Field studies
+Studying trajectory data from railway stations, which were kindly provided by Swiss Federal Railways (SBB AG), it became clear that the variability of types of users is important when describing how the space at railway platforms is used. While alighting passengers mostly leave the platform directly, boarding passengers spend a significant amount of time waiting. Therewith their trajectories cover a larger area of the platform and exhibit certain waiting places which can be identified by 'knots' in the trajectories.
+
+{{< figure src="traj_einaus.png" caption="Trajectories of boarding (red) and alighting passengers (blue) at Zürich Hardbrücke, Switzerland" >}}
+The arrival of trains structures the space usage at the platform in distinct phases, which can be described by waiting or boarding and alighting. At a two-sided platform these phases can also overlap. Passengers mainly use the side of the platform where the train they intend to board is scheduled to arrive, even if the occupation and density on the other side are much lower. Therefore, both platform sides should be analysed separately. 
+
+Additionally, the presence of waiting passengers causes the distribution of mean speeds to show a double peak structure: one peak at lower speeds representing the boarding passengers, the second peak corresponding to alighting and moving passengers. Details can be found in [[1]](#Küpper1).
+
+{{< figure src="meanspeed.png" caption="a) Mean speed of all passengers during afternoon peak hours in Bern, Switzerland; b) mean speed of boarders (red) and alighters (blue)" >}}
+
+### Social Groups and Waiting Time
+In order to analyse the waiting places of passengers and their space occupancy,  in the following investigations a passenger is considered as waiting with speeds less than 0.4 m/s. This threshold was determined through the mean speed distributions of boarding and alighting passengers.  
+
+Previous research in the field of pedestrian dynamics highlighted the importance to consider differences between social groups and individuals, and also introduced methods to detect those in trajectory data. In the context of train stations, we found a lower passenger amount on weekends but a higher percentage of social group members than on working days. Independent of the day of the week, the most frequent group size were pairs. While individual passengers often wait at the sides of stairs and obstacles, social groups with a higher number of members are more likely to wait in front of the platform's entrances. Social groups position themselves in a way that ensures the groups communication, mainly in circles, which results in higher space requirements. 
```

Comment:
Again: It would be nice to know the data basis which you analyzed for that purpose - how many hours/days/weeks